src/web/mjs/connectors/FrScan.mjs

```diff
@@ -0,0 +1,31 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class FrScan extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'fr-scan';
```

Comment:
dont put '-' in id. Bad idea. 

use the same than me https://github.com/manga-download/haruneko/blob/master/web/src/engine/websites/FRTiretScan.ts, for portability: `frtiretscan`